src/View/View/ClientView.java

```diff
@@ -13,8 +16,9 @@ public void PayForOrder() {
 	}
 
 	public void ValuateOrder() {
-		// TODO - implement ClientView.ValuateOrder
-		throw new UnsupportedOperationException();
+		Presenter.Presenter.IPresenter presenter = new Facade();
+		GetValuationRequest request = new GetValuationRequest(); //No attributes inside
```

Comment:
widać że nie ma atrybutów - jak już to napisz dlaczego albo całkiem usuń komentarz

Reply:
albo dodaj 
```suggestion
		var request = new GetValuationRequest(); //TODO implement attributes in future
```